src/srmouse/viz.py

```diff
@@ -42,6 +42,7 @@ def plotReflectanceSignature(df:pd.DataFrame, xlabel:str="Bands (nm)", ylabel:st
                bandNames)
     plt.title(title)
     plt.tight_layout()
+    # plt.savefig("Reflectance_Signature_results.png", dpi=150)  # Uncomment this line if you want to save your plot as .png-file
```

Comment:
Please add this options in the parameters of the method as I requested in a previous review